packages/ui/src/components/molecules/Table/AllocationShare.tsx

```diff
@@ -0,0 +1,56 @@
+import React, { FC } from 'react';
+import styled from 'styled-components';
+
+import { NameVariants } from './IconNameBox';
+
+import { Typography } from '../../atoms';
+
+interface AllocationShareProps {
+  percentage: number;
+  variant: NameVariants;
+  size?: 'small' | 'big';
+}
+
+const assetColorMap: Record<NameVariants, string> = {
+  Bitcoin: '#F89C2D',
+  Ethereum: '#0085FF',
+};
```

Comment:
`packages/ui` should not have any knowledge of the coins. Get these colors from `props`